PKGBUILD

```diff
@@ -1,31 +1,41 @@
 # Maintainer: Thor77 <thor77 at thor77 dot org>
+# Contributor: Sebastian Krebs <sebastian[at]krebs[dot]one>
 
 pkgname=telegraf-bin
 pkgver=1.1.2
 pkgrel=1
```

Comment:
You should increase the package version. I guess I missed that in my patch too...